packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -474,3 +515,27 @@ class VideoPlayerWebOptionsControls {
     return controlsList.join(' ');
   }
 }
+
+/// [PictureInPictureOverlaySettings] can be optionally used to set the position and size of the picture-in-picture overlay
+@immutable
+class PictureInPictureOverlaySettings {
+  /// set the position and size of the picture-in-picture overlay
+  const PictureInPictureOverlaySettings({
+    required this.top,
+    required this.left,
+    required this.width,
+    required this.height,
```

Comment:
This should be a `Rect`, rather than four separate properties. And per my comments above, the exact meaning of the rect (coordinate space, pixel type) needs to be documented.

Reply:
I have refactorred it to use a rect. But regarding the coordinate space, pixel type I have no idea what I should be using here. 

```dart
              final RenderBox? box =
                  _playerKey.currentContext?.findRenderObject() as RenderBox?;
              if (box == null) {
                return;
              }
              final Offset offset = box.localToGlobal(Offset.zero);
              _controller.setPictureInPictureOverlaySettings(
                settings: PictureInPictureOverlaySettings(
                  rect: Rect.fromLTWH(
                    offset.dx,
                    offset.dy,
                    box.size.width,
                    box.size.height,
                  ),
                ),
              );
```

this is how you should call it

Reply:
`The rect represents the global Flutter coordinates using logic pixels of the picture-in-picture overlay.`